Steam Web Integration.user.js

```diff
@@ -192,9 +193,9 @@ function init(userdata, decommissioned, barter) {
         GM_setValue("swi_last", lastCached);
         GM_setValue("swi_data", JSON.stringify(userdata));
     }
-    const lcs = (new Date(lastCached)).toLocaleString();
-    const blcs = (new Date(GM_getValue("swi_barter_last", 0))).toLocaleString();
-    const dlcs = (new Date(GM_getValue("swi_decommissioned_last", 0))).toLocaleString();
+    const lcs = dateOverride ? (new Date(lastCached)).toLocaleString("sv-SE") : (new Date(lastCached)).toLocaleString();
+    const blcs = dateOverride ? (new Date(GM_getValue("swi_barter_last", 0))).toLocaleString("sv-SE") : (new Date(GM_getValue("swi_barter_last", 0))).toLocaleString();
+    const dlcs = dateOverride ? (new Date(GM_getValue("swi_decommissioned_last", 0))).toLocaleString("sv-SE") : (new Date(GM_getValue("swi_decommissioned_last", 0))).toLocaleString();
```

Comment:
Sweden's date format is officially _not dumb_ - but [everybody else has bad ones](https://help.salesforce.com/articleView?id=admin_supported_locales.htm&type=5).